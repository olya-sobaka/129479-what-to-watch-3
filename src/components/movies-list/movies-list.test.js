import React from "react";
import renderer from "react-test-renderer";
import MoviesList from "./movies-list.js";

const movies = [
  {
    title: `Kiki's Delivery Service`,
    posterSrc: `img/midnight-special.jpg`,
    id: 36
  },
  {
    title: `Kiki's Delivery Service`,
    posterSrc: `img/midnight-special.jpg`,
    id: 37
  },
  {
    title: `Kiki's Delivery Service`,
    posterSrc: `img/midnight-special.jpg`,
    id: 38
  }
];

it(`MoviesList renders correctly`, () => {
  const tree = renderer.create(
      <MoviesList films={movies} />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
