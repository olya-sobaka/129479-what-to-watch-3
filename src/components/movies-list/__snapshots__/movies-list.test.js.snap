// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MoviesList renders correctly 1`] = `
<div
  className="catalog__movies-list"
>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Kiki's Delivery Service"
        height="175"
        src="img/midnight-special.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        Kiki's Delivery Service
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Kiki's Delivery Service"
        height="175"
        src="img/midnight-special.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        Kiki's Delivery Service
      </a>
    </h3>
  </article>
  <article
    className="small-movie-card catalog__movies-card"
    onMouseEnter={[Function]}
    onMouseOut={[Function]}
  >
    <div
      className="small-movie-card__image"
    >
      <img
        alt="Kiki's Delivery Service"
        height="175"
        src="img/midnight-special.jpg"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <a
        className="small-movie-card__link"
      >
        Kiki's Delivery Service
      </a>
    </h3>
  </article>
</div>
`;
